api/src/routers/meal_get.js

```diff
@@ -0,0 +1,90 @@
+import "dotenv/config";
+import express from "express";
+import knex from "../database_client.js";
+
+
+const mealRouter = express.Router()
+
+
+mealRouter.get("/future-meals", async (req, res) => {
+    try {
+      const SHOW_TABLES_QUERY =
+        process.env.DB_CLIENT === "mysql2"
+          ? "SELECT * FROM Meal WHERE Meal.When > NOW()"
+          : "SHOW TABLES;";
```

Comment:
```suggestion
      const FUTURE_MEALS_QUERY = "SELECT * FROM Meal WHERE Meal.When > NOW()";
```

The example `SHOW_TABLES_QUERY` was just an example query, you do not have to keep it as a functionality.

Reply:
Great point
i thought we had to use process.env.db_client
thanks for pointing it out